### PR TITLE
fasttext 0.2.0 (new formula)

### LINKED ISSUE #429: Can somebody provide a brew install for fastText?
It would be convenient if fastText can be installed with homebrew. Can somebody provide a brew install for fastText?

### PR DESCRIPTION
**Pull request by sakjur**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
Fixes: facebookresearch/fastText#429

### CODE REVIEW

File: Formula/fasttext.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"trainingset").write("__label__brew brew")
+    system "#{bin}/fasttext", "supervised", "-input", "trainingset", "-output", "model"
```
Comment (fxcoudert) [inline Formula/fasttext.rb]: Can we check that the output file has been created?

File: Formula/fasttext.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+class Fasttext < Formula
+  desc "Library for fast text representation and classification"
+  homepage "https://fasttext.cc"
+  url "https://github.com/facebookresearch/fastText/archive/v0.2.0.tar.gz"
+  sha256 "71d24ffec9fcc4364554ecac2b3308d834178c903d16d090aa6be9ea6b8e480c"
+  head "https://github.com/facebookresearch/fastText.git"
+  depends_on "cmake" => :build
```
Comment (fxcoudert) [inline Formula/fasttext.rb]:
```suggestion

  depends_on "cmake" => :build
```

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/fasttext.rb (hunk 1 above)
Comment (sakjur) [reply Formula/fasttext.rb]: Verified that the model is created in c091b4d

**Comment by sakjur** [comment]
@fxcoudert Not sure what is considered best practice in Homebrew, should I squash the commits before a second review?

**Comment by fxcoudert** [comment]
Yes please squash the commits into a single on, with commit message `fasttext 0.2.0 (new formula)`

**Review by fxcoudert — APPROVED** [review]

**Comment by sakjur** [comment]
✅ Let me know if there's anything else I'll have to do 🙂 Thanks for the review!

**Comment by fxcoudert** [comment]
@sakjur thanks for the pull request, it's all good and I'm merging!

**Pull request closed without merging**